galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/docker/local/DockerLocalJava08Ubuntu.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.inttests.docker.local;
+
+import dev.galasa.BeforeClass;
+import dev.galasa.Test;
+import dev.galasa.TestAreas;
+import dev.galasa.galasaecosystem.IGenericEcosystem;
+import dev.galasa.galasaecosystem.ILocalEcosystem;
+import dev.galasa.galasaecosystem.LocalEcosystem;
+import dev.galasa.inttests.docker.AbstractDockerLocal;
+import dev.galasa.ipnetwork.ICommandShell;
+import dev.galasa.java.JavaVersion;
+import dev.galasa.java.ubuntu.IJavaUbuntuInstallation;
+import dev.galasa.java.ubuntu.JavaUbuntuInstallation;
+import dev.galasa.linux.ILinuxImage;
+import dev.galasa.linux.LinuxImage;
+import dev.galasa.linux.OperatingSystem;
+
+@Test
+@TestAreas({"dockermanager", "localecosystem", "java08", "ubuntu"})
+public class DockerLocalJava08Ubuntu extends AbstractDockerLocal {
+	
+	@LocalEcosystem(linuxImageTag = "PRIMARY")
+	public ILocalEcosystem ecosystem;
+	
+	@LinuxImage(operatingSystem = OperatingSystem.ubuntu)
```

Comment:
Needs a capability of "nonshared" so a standalone instance is requested
And needs a different tag as you don't want the local ecosystem installing on the same server.